eZ/Publish/API/Repository/Values/ContentType/ContentType.php

```diff
@@ -179,13 +138,6 @@
     protected $isContainer;
 
     /**
-     * Main language.
-     *
-     * @var string
-     */
-    protected $mainLanguageCode;
```

Comment:
ok, yeah I see it is not there on the other domain types which now uses these traits, so +1 then